packages/itwinui-react/src/core/Tabs/Tabs.tsx

```diff
@@ -451,36 +455,116 @@ export const Tabs = (props: TabsProps) => {
     ownerDoc.addEventListener('scroll', determineScrollingPlacement);
   }, [overflowOptions?.useOverflow, determineScrollingPlacement]);
 
-  const onTabClick = React.useCallback(
-    (index: number) => {
-      if (onTabSelected) {
-        onTabSelected(index);
-      }
-      setCurrentActiveIndex(index);
-    },
-    [onTabSelected],
+  return (
+    <Box
+      className={cx(
+        'iui-tabs',
+        `iui-${type}`,
+        {
+          'iui-green': color === 'green',
+          'iui-animated': type !== 'default' && isClient,
+          'iui-not-animated': type !== 'default' && !isClient,
+          'iui-large': hasSublabel,
+        },
+        className,
+      )}
+      data-iui-overflow={overflowOptions?.useOverflow}
+      data-iui-scroll-placement={scrollingPlacement}
+      role='tablist'
+      ref={refs}
+      {...rest}
+    >
+      <TabListContext.Provider
+        value={{
+          onTabClick,
+          setFocusedValue,
+          focusActivationMode,
+          hasSublabel,
+          setHasSublabel,
+        }}
+      >
+        {children}
+      </TabListContext.Provider>
+    </Box>
   );
+}) as PolymorphicForwardRefComponent<'div', TabListOwnProps>;
+TabList.displayName = 'Tabs.TabList';
+
+// ----------------------------------------------------------------------------
+// Tabs.Tab component
+
+type TabOwnProps = {
+  /**
+   * Value used to associate the tab with a given panel.
+   */
+  value: string;
+  /**
+   * Tab label used for simple Tab construction.
+   * Cannot be used with tabs that have icons or descriptions.
+   */
+  label?: string | React.ReactNode;
+  /**
+   * Flag whether the tab is active.
+   * @default 'false'
+   */
+  isActive?: boolean;
+  /**
+   * Callback fired when the isActive prop changes.
+   */
+  onActivated?: () => void;
```

Comment:
bikeshed: `onActivated` vs `onActiveChange`?

for comparison: Popover and Tooltip have `onVisibleChange`

Reply:
Changed to onActiveChanged. Sounds better.